lib/http/__init__.py

```diff
@@ -89,6 +89,12 @@
 # send/receive quantum
 SOCK_BUF_SIZE = 32768
 
+# OpenSSL.SSL.ConnectionType was deprecated in pyopenssl-19.1.0:
+try:
+    ssl_conn_type = OpenSSL.SSL.Connection
+except AttributeError:
+    ssl_conn_type = OpenSSL.SSL.ConnectionType
```

Comment:
This is basically a constant, so I'd suggest instead using `SSL_CONN_TYPE`, not lowercase version.

Reply:
I've made a new commit that does this.